admin_manual/maintenance/update.rst

```diff
@@ -145,20 +145,49 @@ user is::
 
     chown -R <http-user>:<http-user> /path/to/owncloud/
 
-* This example is for Ubuntu 14.04 LTS server::
-   
-    chown -R www-data:www-data /var/www/owncloud
-
-* Arch Linux::
-
-    chown -R http:http /path/to/owncloud/
+You can find your HTTP user in your HTTP server configuration files. Or you can 
+use :ref:`label-phpinfo` (Look for the **User/Group** line).
+
+* The HTTP user and group in Debian/Ubuntu is ``www-data``.
+* The HTTP user and group in Fedora/CentOS is ``apache``.
+* The HTTP user and group in Arch Linux is ``http``.
+* The HTTP user in openSUSE is ``wwwrun``, and the HTTP group is ``www``.
+
+The easy way to set the correct permissions is to copy and run the following scripts.
+The first script sets the permissions for the update process, the second script
+resets the permissions back to the strict settings described above. 
+Replace the ``ocpath`` variable with the path to your ownCloud directory, and 
+replace the ``htuser`` and ``htgroup`` variables with your HTTP user and group.
+
+Script preparing the instance for upgrading
+-------------------------------------------
+  ::
+  
+    #!/bin/bash
+    # Sets permissions of the owncloud instance for updating
+    
+    ocpath='/var/www/owncloud'
+    htuser='www-data'
+    htgroup='www-data'
+    rootuser='root' # On QNAP this is admin
+    
+    # If the script from the Installation Wizard section:
+    # "Setting Strong Directory Permissions" has been used, you do not need to reset
+    # the permissions on /apps, /config, /data and /themes
+    # as they already should be correct (${htuser}:${htgroup})
```

Comment:
Yes, you commented, but you may got confused.
I try to give you an explanation:

- when you install you should set strong permissions for security reasons. Described in install_wizard.rst
- when you want to upgrade via the updater app, these strong permissions hinder the app from processing. Therefore you need to set the permissions to a different level (full write access to the webserver user). Note, some directories already have the correct permissions. 
- when the upgrade has been finised, you need to reset the permissions back to strong

So what is to do now:
- before you update, call the script I added as described here in update.rst (Script preparing the instance **for** upgrading)
- do the upgrade via the updater app
- call the script from the installation wizard again to reset to strong permissions. This is also described in update.rst (Script preparing the instance **after** upgrading)

I hope that this description and my PR makes sense to you.